app/services/push_notification/http_push.rb

```diff
@@ -38,6 +38,8 @@ def url_options
     def deliver_one(service_provider)
       if AppConfig.env.risc_notifications_sqs_enabled == 'true'
         deliver_sqs(service_provider)
+      elsif IdentityConfig.store.risc_notifications_local_enabled
+        deliver_local(service_provider)
```

Comment:
what if we made this an "and" instead of an or? ex deliver to local dashboard *and* store locally?

Reply:
I like that a lot, added in https://github.com/18F/identity-idp/pull/4918/commits/4e7d7416b41245bfd8426d826f08b3a6e9d5f4ed